test.rb

```diff
@@ -1,6 +1,6 @@
 class TestClass
 
 	def foo
-	puts 'bar'
+		puts 'bar'
```

Comment:
Use 2 (not 1) spaces for indentation.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Tab detected.